packages/atomic/src/global/global.pcss

```diff
@@ -53,7 +53,7 @@
   }
 
   .btn-pill {
-    @apply rounded-2xl;
+    @apply rounded-xl;
```

Comment:
Should this class be in `global.pcss`? Or can it be in `pcss` file for the breadbox?

Reply:
It's generic and reusable, but it's true that right now only one component uses it